src/backend/distributed/executor/executor.c

```diff
@@ -2465,7 +2469,7 @@ WorkerPoolFailed(WorkerPool *workerPool)
 	ListCell *sessionCell = NULL;
 
 	/* a pool cannot fail multiple times */
-	Assert (!workerPool->failed);
+	Assert(!workerPool->failed);
```

Comment:
should maybe be in separate commits.

Reply:
Just removed those, I'll have another PR for indent etc.